tests/e2e/specFiles/addressBook.spec.ts

```diff
@@ -61,15 +61,15 @@ describe('Address book', () => {
 
       it("shouldn't be possible to create a new contact if name is invalid", async () => {
         await app.addressBook.contactsScreen.tapOnCreateButton();
-        await app.addressBook.newContact.addNewContactScreen.typeName('Foo-name!');
-        await app.addressBook.newContact.addNewContactScreen.submit();
+        await app.addressBook.newContact.addNewContactScreen.typeName('Foo-name!;');
         await waitFor(app.addressBook.newContact.addNewContactScreen.nameValidationError)
           .toBeVisible()
           .withTimeout(WAIT_FOR_ELEMENT_TIMEOUT.DEFAULT);
       });
 
       it("shouldn't be possible to create a new contact if address is invalid", async () => {
         await app.addressBook.contactsScreen.tapOnCreateButton();
+        await app.addressBook.newContact.addNewContactScreen.typeName(`Heisenberg`);
         await app.addressBook.newContact.addNewContactScreen.typeAddress('fOoBa5bAZ');
         await app.addressBook.newContact.addNewContactScreen.submit(); // TODO: Remove it once it's fixed in the app
```

Comment:
this comment we need still keep ?

Reply:
As discussed on the daily, I am going to resolve todos later. However in this particular case the test case works as it is, I don't know what's the fix supposed to be - there is no reference to any Jira ticket. I guess it's about the address format being validated by the form, before submission, but that's just a guess.